luarules/gadgets/unit_paralyze_damage_limit.lua

```diff
@@ -47,8 +47,9 @@ for udid, ud in pairs(UnitDefs) do
 
 	
 	if modOptions.emprework==true then
-		unitOhms[udid] = ud.customParams.paralyzemultiplier or 1
-		--Spring.Echo('ohm', ud.customParams.paralyzemultiplier)
+		unitOhms[udid] = ud.customParams.paralyzemultiplier == nil and 1 or tonumber(ud.customParams.paralyzemultiplier)--it arrives as a string because WHY NOT
+		
+		--Spring.Echo('ohm', ud.customParams.paralyzemultiplier, unitOhms[udid])
 		if tonumber(ud.customParams.paralyzemultiplier) or 0 > 0 then
 		--Spring.Echo('ohm', ud.name, ud.customParams.paralyzemultiplier)
 		end
```

Comment:
✂️

Reply:
conditional debugging, but removed - can't promise it won't be back next time I'm chasing bugs around